pkg/varlinkapi/util.go

```diff
@@ -133,3 +136,33 @@ func handlePodCall(call iopodman.VarlinkCall, pod *libpod.Pod, ctrErrs map[strin
 
 	return nil
 }
+
+func stringCompressionToArchiveType(s string) archive.Compression {
+	if len(s) == 0 {
```

Comment:
This would be cleaner if you just made this the default. Or just remove this block.  It will fall back to the same.